src/MSBuild.SDK.SystemWeb.RazorLibrary/Sdk/MSBuild.SDK.SystemWeb.RazorLibrary.DefaultPackages.targets

```diff
@@ -1,20 +1,82 @@
+    *  CentralPackageManagement (NuGet Capability)
+        - https://github.com/CZEMacLeod/MSBuild.SDK.SystemWeb/issues/46
+        - https://github.com/CZEMacLeod/MSBuild.SDK.SystemWeb/issues/49
+        - Look for the property "ManagePackageVersionsCentrally" 
+        
+  *************************************************************************************************************************
+  -->
+  <ItemGroup Condition=" '$(ExcludeDefaultRazorPackages)'=='false' ">
+
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset"
+                      Condition="'$(MicrosoftNetCompilersToolset_Version)'!=''"
+                      Exclude="@(PackageReference)" >
+      <SystemWebSdkIncludedPackage>true</SystemWebSdkIncludedPackage>
+      <SystemWebSdkIncludedPackageVersion>$(MicrosoftNetCompilersToolset_Version)</SystemWebSdkIncludedPackageVersion>
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
+
+    <PackageReference Include="Microsoft.CodeDom.Providers.DotNetCompilerPlatform"
+                      Condition="'$(MicrosoftCodeDomProvidersDotNetCompilerPlatform_Version)'!=''"
+                      Exclude="@(PackageReference)" >
+      <SystemWebSdkIncludedPackage>true</SystemWebSdkIncludedPackage>
+      <SystemWebSdkIncludedPackageVersion>$(MicrosoftCodeDomProvidersDotNetCompilerPlatform_Version)</SystemWebSdkIncludedPackageVersion>
+    </PackageReference>
+
+    <PackageReference Include="RazorGenerator.MsBuild"
+                      Condition="'$(RazorGeneratorMSBuild_Version)'!=''"
+                      Exclude="@(PackageReference)" >
+      <SystemWebSdkIncludedPackage>true</SystemWebSdkIncludedPackage>
+      <SystemWebSdkIncludedPackageVersion>$(RazorGeneratorMSBuild_Version)</SystemWebSdkIncludedPackageVersion>
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
+
+    <PackageReference Include="Microsoft.AspNet.Mvc"
+                      Condition="'$(MicrosoftAspNetMvc_Version)'!=''"
+                      Exclude="@(PackageReference)" >
+      <SystemWebSdkIncludedPackage>true</SystemWebSdkIncludedPackage>
+      <SystemWebSdkIncludedPackageVersion>$(MicrosoftAspNetMvc_Version)</SystemWebSdkIncludedPackageVersion>
+    </PackageReference>
+
+    <PackageReference Include="RazorGenerator.MVC"
+                      Condition="'$(RazorGeneratorMVC_Version)'!=''"
+                      Exclude="@(PackageReference)" >
+      <SystemWebSdkIncludedPackage>true</SystemWebSdkIncludedPackage>
+      <SystemWebSdkIncludedPackageVersion>$(RazorGeneratorMVC_Version)</SystemWebSdkIncludedPackageVersion>
+    </PackageReference>
+
+    
+    <PackageReference Update="@(PackageReference->WithMetadataValue('SystemWebSdkIncludedPackage','true')->HasMetadata('SystemWebSdkIncludedPackageVersion'))"
+                      Condition=" '$(UsingMicrosoftCentralPackageVersionsSdk)' != 'true' AND '$(ManagePackageVersionsCentrally)' != 'true' ">
```

Comment:
Can use `'$(ApplySDKDefaultPackageVersions)'=='true'` as the condition as of V4.0.88
This should probably be in the common files target now.